src/components/inputs/InputLabel.js

```diff
@@ -52,20 +22,16 @@ function InputLabelComponent({
     );
 }
 InputLabelComponent.defaultProps = {
-    isFocused: false,
```

Comment:
How come we don't need that prop anymore?
If so, we will need to clean up Iaso, where it's probably a state value in the parent

Reply:
cf: "// TODO investigate isFocused prop. It doesn't seem to do anything"  in deprecated inputComponent. This the only one

Reply:
That looks like something I wrote 😅